lib/parse-model.js

```diff
@@ -33,16 +33,33 @@ function parseModel(model) {
 
 
 /**
- * parseArray - Given a model and options, iterates 'size' times to generate
- * a list of the parsed 'model' type.
+ * parseArray - Takes a model, options, and size. If the size is an object (ex. [1, 20])
+ * it will use the randomBetween method from numbers to get a random number between
+ * the first and second index.
+ * If size is a simple number, it just uses that.
  * -> parseModel can be either a Node or a Leaf.
  *
  * @param  {Object} model   A model Node
  * @param  {Object} options [size: Number]
  * @return {Array} A parsed model
  */
 function parseArray(model, options) {
-  return [...Array(options.size).keys()].map(() => parseModel(model.value));
+  let len;
+  let size = options.size;
+  if (typeof size === "object") {
```

Comment:
What do you think about using `Array.isArray` instead of typeof ? I've got the feeling `object` might be a little too generic.

Reply:
Makes sense. Updated :)